src/client/containers/authentication/authentication.css

```diff
@@ -0,0 +1,80 @@
+.authentication {
+    font-family: var(--font-authentication);
+    color: var(--authFont-white);
+    background-image: url('../../assets/images/authBackground.jpeg');
```

Comment:
Compare these rules using the short form)

Reply:
done